api_types.ts

```diff
@@ -775,6 +776,11 @@ export interface ExecutionContext {
    * Information about state of the current sync. Only populated if this is a sync table formula.
    */
   readonly sync?: Sync;
+
+  /**
+   * {@link core.MetadataContext}
+   */
+  parameters?: MetadataContext;
```

Comment:
looks like everything else is readonly here... any reason not to put that on this too?

Reply:
oh yeah this should e readonly too